cpp/ql/src/Diagnostics/FailedCompilations.ql

```diff
@@ -0,0 +1,22 @@
+/**
+ * @name Failed compiler invocations.
```

Comment:
```suggestion
 * @name Failed compiler invocations
```

Should it say "compiler" here? The alert message says "Extraction failed", which seems correct.